src/batch-client/index.ts

```diff
@@ -241,7 +241,7 @@ export class ZimbraBatchClient {
 				...denormalize(CalendarItemCreateModifyRequest)(appointment)
 			},
 			accountName: accountName
-		}).then(({ invId }) => ({ inviteId: invId, created: !!invId }));
+		}).then(({ invId }) => ({ inviteId: invId }));
```

Comment:
instead of returning only inviteId you should return all data which is available in CalendarItemHitInfo type